release/models/vlan/openconfig-vlan.yang

```diff
@@ -140,7 +161,15 @@ module openconfig-vlan {
     description
       "TPID opstate for dot1q-enabled interfaces";
 
-    // placeholder
+    leaf tpid {
```

Comment:
As above, I think this is already defined based on it being within the `config` container.

Reply:
OK